### ISSUE TITLE
Ensure ARIA attributes are not prohibited for an element's role

### ISSUE THREAD

**Issue by OlgaLarina** [surveyjs/survey-creator#6928]
<span class="svc-item-value-controls__button svc-item-value-controls__remove" aria-label="Click to remove the item..." tabindex="0"><svg class="sv-svg-icon" role="presentation"><use xlink:href="#icon-remove-16x16"></use><title>Click to remove the item...</title></svg></span>

aria-label attribute cannot be used on a span with no valid role attribute.

![Image](https://github.com/user-attachments/assets/ba4abb18-58d2-4e45-b2fe-d2c36967a494)
